CodeEdit/Features/QuickOpen/ViewModels/QuickOpenViewModel.swift

```diff
@@ -42,24 +42,28 @@ final class QuickOpenViewModel: ObservableObject {
                     .isRegularFileKey
                 ],
                 options: [
-                    .skipsHiddenFiles,
                     .skipsPackageDescendants
                 ]
             )
             if let filePaths = enumerator?.allObjects as? [URL] {
-                let files = filePaths.filter { url in
-                    let state1 = url.lastPathComponent.lowercased().contains(self.openQuicklyQuery.lowercased())
+                /// removes all filePaths which aren't regular files
+                let filteredFiles = filePaths.filter { url in
                     do {
                         let values = try url.resourceValues(forKeys: [.isRegularFileKey])
-                        return state1 && (values.isRegularFile ?? false)
+                        return (values.isRegularFile ?? false) && !url.absoluteString.contains(".git")
```

Comment:
Don't exclude the git directory like this. It shouldn't contain any files that might mess up a search and it'll be clear in the search results that they are from the git directory.

Reply:
At least don't do this manually. We might include a setting at some point that excludes specified things from search results.

Reply:
OK no problem. I just figured the user shouldn't see something like this:
<img width="1119" alt="Screenshot 2023-05-29 at 6 52 05 PM" src="https://github.com/CodeEditApp/CodeEdit/assets/83090745/59f56483-551d-4b58-9962-842ad7e3bf6e">